addon/components/polaris-button-group.js

```diff
@@ -67,13 +67,14 @@ export default Component.extend({
     this._super(...arguments);
 
     // Wrap each child element that isn't already a group item.
+    let nodesToWrap = childrenWithoutClassName(
+      this.element.children,
```

Comment:
The name `childrenWithoutClassName` implies to me that the function will take a node, and return a list of its children filtered by class name - but here it looks like it's actually returning a filtered copy of the passed-in list of nodes and not accessing any children. If that's the case, can we update the name to remove the `children` part - maybe something like `rejectNodesByClassName`?

Reply:
also works for me 😛